tests/cpptests/test_cpp_iterator_index.cpp

```diff
@@ -6,65 +6,64 @@
 
 #include "gtest/gtest.h"
 #include "iterator_util.h"
-#include "index_utils.h"
+#include <vector>
 
+extern "C" {
 #include "src/forward_index.h"
 #include "src/iterators/inverted_index_iterator.h"
+#include "src/redis_index.h"
+#include "src/spec.h"
+#include "src/index_result.h"
+#include "src/util/dict.h"
+#include "src/tag_index.h"
+#include "src/numeric_index.h"
+#include "redisearch_rs/headers/triemap.h"
+}
 
-typedef enum IndexIteratorType {
-    TYPE_TERM_FULL,
-    TYPE_NUMERIC_FULL,
-    TYPE_TERM,
-    TYPE_NUMERIC,
-    TYPE_GENERIC,
-} IndexIteratorType;
+typedef enum IndexType {
+    INDEX_TYPE_TERM_FULL,
+    INDEX_TYPE_NUMERIC_FULL,
+    INDEX_TYPE_TERM,
+    INDEX_TYPE_NUMERIC,
+    INDEX_TYPE_GENERIC,
+} IndexType;
 
-class IndexIteratorTest : public ::testing::TestWithParam<std::tuple<IndexIteratorType, bool>> {
+class IndexIteratorTest : public ::testing::TestWithParam<IndexType> {
```

Comment:
Seems like a rebase was deleting all the TTL variant tests. Need to revert

Reply:
added back